e2e/pages/adf/settingsPage.ts

```diff
@@ -96,9 +98,43 @@ export class SettingsPage {
         return this;
     }
 
-    clickApply() {
+    async clickSsoRadioButton () {
+        Util.waitUntilElementIsVisible(this.ssoRadioButton);
+        await this.ssoRadioButton.click();
+    }
+
+    async setSSO (processServiceURL, authHost ) {
+        this.goToSettingsPage();
+        this.setProvider(this.bpm.option, this.bpm.text);
+        Util.waitUntilElementIsVisible(this.bpmText);
+        expect(this.ecmText.isPresent()).toBe(false);
+        await this.clickSsoRadioButton();
+        await this.setProcessServicesURL(processServiceURL);
+        await this.setAuthHost(authHost);
+        await this.disableSilentLogin();
+        await this.clickApply();
+    }
+
+    async setProcessServicesURL (processServiceURL) {
+        Util.waitUntilElementIsVisible(this.bpmText);
+        await this.bpmText.clear();
+        await this.bpmText.sendKeys(processServiceURL);
+    }
+
+    async setAuthHost (authHostURL) {
+        Util.waitUntilElementIsVisible(this.authHostText);
+        await this.authHostText.clear();
+        await this.authHostText.sendKeys(authHostURL);
+    }
+
+    async clickApply () {
         Util.waitUntilElementIsVisible(this.applyButton);
-        this.applyButton.click();
+        await this.applyButton.click();
+    }
+
+    async disableSilentLogin () {
```

Comment:
Please rename disableSilentLogin method to setSilentLogin(). Also this method should contain a parameter. Because the param can be optional, we should use a default value.[in our case false for silentLogin and true for Implicitflow is the best]